ghmirror/core/mirror_requests.py

```diff
@@ -58,6 +58,68 @@ def _cache_response(resp, cache, cache_key):
         cache[cache_key] = resp
 
 
+def _online_request(method, url, cached_response,
+                    headers=None, parameters=None):
+    """
+    Handle API errors on conditional requests and try
+    to serve contents from cache
+    """
+    try:
+        resp = requests.request(method=method,
+                                url=url,
+                                headers=headers,
+                                timeout=REQUESTS_TIMEOUT,
+                                params=parameters)
+
+        # When we hit the API limit, or there is a problem with the API
+        # let's try to serve from cache
+        response_header = _should_error_response_be_served_from_cache(resp)
```

Comment:
```suggestion
        error_response_header = _should_error_response_be_served_from_cache(resp)
```
This will make things clearer as to why we return if it is `None`